extensions/blocks/podcast-player/style.scss

```diff
@@ -118,6 +118,18 @@ $player-float-background: $light-gray-200;
 		}
 	}
 
+	// Apply `secondary` color to the podcast title.
+	.jetpack-podcast-player__container:not(.has-secondary) { // no custom -> default color.
+		a.jetpack-podcast-player__podcast-title {
+			color: $text-color;
+		}
+	}
+	.jetpack-podcast-player__container.has-secondary { // custom color.
+		a.jetpack-podcast-player__podcast-title {
```

Comment:
I think you need to remove the `a` before these class names, as it's possible for these to _not_ be a link.

Reply:
well this is how it's applied right now.

Reply:
Not anymore :) I just removed it since it was being repeated.